ros2pkg/ros2pkg/package_type/__init__.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020 Wayne Parrott
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ros2cli.entry_points import get_entry_points
+from ros2cli.plugin_system import _instantiate_extension
```

Comment:
The underscore prefix indicates that the function is considered non-public. So you shouldn't use since it can change anytime in any way without notice.

Reply:
I meant to call out that I used of an internal function to avoid instantiating all package_type extensions. The objective is to instantiate a single package_type by name. ros2cli.plugin_system provides us only 2 functions for extension instantiation:
```
def instantiate_extensions(group_name, *, exclude_names=None, unique_instance=False)

def _instantiate_extension(group_name, extension_name, extension_class, *, unique_instance=False)
```
Being aware that use of internal functions is considered bad form, I initially used the heavier `instantiate_extensions()` which generated feedback to be more efficient and avoid instantiating all package_type extensions just to access the extension of interest. So my plan-B resulted in use of the internal function `_instantiate_extension(ext_name)`. I'm not aware of other api's that I can call on that will provide me what _instantiate_extension() does.

I recognize the big push for the upcoming release. But if you have some insight into other approaches please push back. My blink recommendation is to simply make _instantiate_extension() public. 

Thoughts?